aws/resource_aws_guardduty_test.go

```diff
@@ -10,6 +10,10 @@ func TestAccAWSGuardDuty(t *testing.T) {
 			"basic":  testAccAwsGuardDutyDetector_basic,
 			"import": testAccAwsGuardDutyDetector_import,
 		},
+		"Member": {
+			"basic":  testAccAwsGuardDutyMember_basic,
+			"import": testAccAwsGuardDutyMember_import,
+		},
```

Comment:
I assume there can be more than 1 member per AWS account, so we don't need to make these tests synchronous? Unless I'm mistaken...

Reply:
This needs to be like this because its like AWS Config, where it can only be activated once in a region, so we're left with serially doing one test at a time since we turn the GuardDuty detector on and off each test.

Reply:
I see, 👌